TASK-5/ShopLogic.cs

```diff
@@ -0,0 +1,66 @@
+using Task5.Utils;
+
+namespace Task5;
+
+public class ShopLogic
+{
+    public const int NumberOfCustomers = 5;
+    private static BogusRepository _repository = new BogusRepository();
+
+    public static List<Ware> CreateWares()
+    {
+        var listOfWares = new List<Ware>();
+        for (var i = 0; i < NumberOfCustomers; i++)
+        {
+            listOfWares.Add(new Ware());
+        }
+
+        var wares = _repository.GetWares(listOfWares);
+
+        return wares;
+    }
+
+    public static List<Cart> CreateEmptyCarts()
+    {
+        var carts = new List<Cart>();
+
+        return carts;
```

Comment:
Delete defining of variable, just return new List<Cart>();

Reply:
Done